common/utilities/CMakeLists.txt

```diff
@@ -13,3 +13,7 @@ target_sources(${LRS_TARGET}
         "${CMAKE_CURRENT_LIST_DIR}/time/l500/get-mfr-ww.h"
         "${CMAKE_CURRENT_LIST_DIR}/time/l500/get-mfr-ww.cpp"
 )
+
+if(WIN32)
```

Comment:
This makefile isn't really win32-specific... shouldn't the ifdef be inside it?

Reply:
done